### ISSUE TITLE
[Enhancement] Modal Size

### ISSUE THREAD

**Issue by maezcruz** [infor-design/enterprise#8098]
Problem:
We noticed that the current size setting of our modal/contextual action panel is good for static content but not for dynamic. In the example below, we used it for search functionality which has different content states such as 'with result' and 'without result'. Is there a way we can have the fixed size for this type of content?

IDS Ticket for reference:
https://jira.infor.com/browse/IDS-1484

**Comment by ericangeles** [comment]
@tmcconechy, let's add this to our upcoming sprint. I spoke with Whey and the team, and this one has a high priority.

**Comment by tmcconechy** [comment]
Sure looks like we have some specs on https://jira.infor.com/browse/IDS-1484
This one may be tricky

**Comment by ericangeles** [comment]
Please feel free to ask if you have any questions, @yohannahbautista.

@tmcconechy, Could you please offer a more comprehensive solution for this enhancement beyond what is outlined in the JIRA ticket?

**Comment by tmcconechy** [comment]
Was this groomed in grooming? I think it may need some discussion. But what im thinking is..

- settings minWidth, width, maxWidth, minHeight, height, maxHeight
- then show two examples where you put in the medium and large buttons with examples of these two modals, the modals literally are those heights/widths as passed. Some of the max/min ones will need to be 100% (minus margins)

This is maybe a tough one to force through without discussing all the interactions. Can it wait to next plan?

**Comment by ericangeles** [comment]
Yes, Tim. We can move this to November sprint.

**Comment by EdwardCoyle** [comment]
Agree, this one has the potential to cause a lot of change to apps with existing modal implementations.  I think it might be best to implement the way Tim describes -- have size settings that can be configured and changed dynamically, but default to the content's size definition (either by the content itself or some CSS rule on the content).

**Comment by talkaboutdesign** [comment]
@tmcconechy I want to park this. We don't have time on design to really define this right now. But we should def look at modals holistically.
